default/templates/layouts/book.tpl

```diff
@@ -70,6 +70,13 @@
           </main>
         </div>
       </div>
+      <style type="text/css">
+        @media (min-width: 768px) {
+          #container {
+            display: flex;
+          }
```

Comment:
Hmm, not sure why this needed. `flex` is already in the class list of `id=container` div further above?

Reply:
that is needed to resolve issue (2). The idea is that when we are on small screen and we click on menu button, then "flex" class is removed from "container" tag. Then, if we change the size to a bigger one so that menu button is hidden, there were no way to put back the "flex" class there were removed as part of clicking on menu button. So, that CSS forces the tag "container" to put back flex mode for that tag and it is showing properly... that is how I am fixing it.

Reply:
You can solve this by switching between `flex-col` and `flex-row` when viewport changes (Tailwind's `md:` thing does it - see https://tailwindcss.com/docs/responsive-design ) ... just pushed a commit.